tests/std/include/test_atomic_wait.hpp

```diff
@@ -38,9 +38,11 @@ void test_atomic_wait_func(const UnderlyingType old_value, const UnderlyingType
         add_seq('4');
         a.store(new_value);
         a.notify_one();
+#ifdef CAN_FAIL_ON_TIMING_ASSUMPTION
         // timing assumption that the main thread evaluates the `wait(old_value)` before this timeout expires
         std::this_thread::sleep_for(waiting_duration);
         add_seq('6');
+#endif

```

Comment:
As this spans more than 1 line:
```suggestion
#endif // CAN_FAIL_ON_TIMING_ASSUMPTION
```